src/cadenza.js

```diff
@@ -844,19 +844,54 @@ export class CadenzaClient {
     return this.#fetch(resolvePath(source), params, signal);
   }
 
+  /**
+   * Fetch object info from a workbook view.
+   *
+   * @param {EmbeddingTargetId} source - The workbook view to fetch object info from.
+   *   Currently only map views are supported.
```

Comment:
Proposal: Change the main description above instead of mentioning this limitation only here. We do not expect the object info to be supported for other views, right?

> Fetch the object info from a workbook map view.